.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  CI: true
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+    steps:
+      - name: Install clang for Windows
+        if: runner.os == 'Windows'
+        run: |
+          iwr -useb get.scoop.sh -outfile 'install.ps1'
+          .\install.ps1 -RunAsAdmin
+          scoop install llvm --global
+
+          # Scoop modifies the PATH so we make the modified PATH global.
+          echo $env:PATH >> $env:GITHUB_PATH
+
+      - name: Fetch code
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        with:
+          fetch-depth: 1
+
+      # Skip macOS & Windows, cache there is slower
```

Comment:
Can you add a reference to this line?

Reply:
this was copy pasted from llhttp. used the standard logic now with setup-node action